src/hooks/useCourtContracts.js

```diff
@@ -564,44 +566,56 @@ export function useActiveBalanceOfAt(juror, termId) {
   return [activeBalance.amount, activeBalance.error]
 }
 
-export function useTotalActiveBalancePolling(termId) {
-  const jurorRegistryContract = useCourtContract(
-    CourtModuleType.JurorsRegistry,
-    jurorRegistryAbi
-  )
-  const [totalActiveBalance, setTotalActiveBalance] = useState(bigNum(-1))
+export function useTotalANTStakedPolling(timeout = 1000) {
+  const [totalANTStaked, setTotalANTStaked] = useState(bigNum(-1))
+  const [error, setError] = useState(false)
+  const [controlledTimeout, setControlledTimeout] = useState(0)
+  const { address: antAddress } = getKnownToken('ANT') || {}
+  const antContract = useContractReadOnly(antAddress, tokenAbi)
 
   useEffect(() => {
     let cancelled = false
     let timeoutId
 
-    const fetchTotalActiveBalance = () => {
+    if (getNetworkType() === 'local') {
+      setError(true)
+      return
+    }
+    if (!antContract) {
+      return
+    }
```

Comment:
I think the first if is not needed. 

What we can do instead is return in the second condition `!antContract` setting the state of `total ANTStaked` to 0

What do you tihnk?

```suggestion
    if (!antContract) {
      return setState(bigNum(0))
    }
```

Reply:
on this one i would preffer to let as it is now since is clear that we are not supporting the local environment